fluent-package/msi/assets/fluent-package-post-migration.bat

```diff
@@ -0,0 +1,46 @@
+@echo on
+title Fluent-package post migration script
+
+if exist "%~dp0..\..\td-agent" (
+  echo "Migrate c:\opt\td-agent\etc\td-agent.conf"
+  if exist "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" (
+     move /Y "%~dp0..\..\td-agent\etc\td-agent\td-agent.conf" "%~dp0..\etc\fluent\td-agent.conf"
+  )
+  echo "Create c:\opt\fluent\var\log\fluent directory"
+  if not exist "%~dp0..\var\log\fluent" (
+     mkdir "%~dp0..\var\log\fluent"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%f "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\var\log\td-agent\* directory (e.g buffer)"
+  for /d %%d in (%~dp0..\..\td-agent\var\log\td-agent\*) do (
+     move /Y %%d "%~dp0..\var\log\fluent\"
+  )
+  echo "Migrate c:\opt\td-agent\etc\td-agent\* files"
+  for %%f in (%~dp0..\..\td-agent\etc\td-agent\*) do (
+     move /Y %%f "%~dp0..\etc\fluent\"
+  )
+  echo "migrate c:\opt\td-agent\etc\plugin directory"
+  for /d %%d in (%~dp0..\..\td-agent\etc\plugin\*) do (
+     move /Y %%d "%~dp0..\etc\fluent\plugin\"
+  )
+  sc query state=inactive | findstr fluentdwinsvc
+  if !ERRORLEVEL! == 1 (

```

Comment:
Shouldn't this be the first process?
(right after the `if exist "%~dp0..\..\td-agent"`)

Reply:
It seems better to change. thanks.